web/src/components/MapOverlays.tsx

```diff
@@ -1,26 +1,35 @@
 import { useFeatureFlag } from 'features/feature-flags/api';
-import { useAtomValue } from 'jotai';
+import { useAtom, useAtomValue } from 'jotai';
 import { lazy, Suspense } from 'react';
 import { useSearchParams } from 'react-router-dom';
-import { hasSeenSurveyCardAtom } from 'utils/state/atoms';
+import {
+  hasSeenSurveyCardAtom,
+  hasSeenUsSurveyCardAtom,
+  userLocationAtom,
+} from 'utils/state/atoms';
 import { useIsBiggerThanMobile } from 'utils/styling';
 
 import SurveyCard from './app-survey/SurveyCard';
 import LegendContainer from './legend/LegendContainer';
 
 export default function MapOverlays() {
   const hasSeenSurveyCard = useAtomValue(hasSeenSurveyCardAtom);
+  const hasSeenUsSurveyCard = useAtomValue(hasSeenUsSurveyCardAtom);
+  const [userLocation] = useAtom(userLocationAtom);
```

Comment:
Any reason why this is not using useAtomValue like the others?

Reply:
Prettier complained when I tried that originally but now it's typed I think it's okay, updated it now.